modules/spock/README.md

```diff
@@ -0,0 +1,48 @@
+# TestContainers-Spock
```

Comment:
No idea why these Spock files are showing up in the PR - looking at the branch locally I don't see this.

Reply:
I am not sure what should I do for this one. Maybe it can be ignored while merging, if appropriate! I don't have any changes for Spock :smile: